src/WebJobs.Script.WebHost/WebScriptHostHttpRoutesManager.cs

```diff
@@ -76,23 +60,46 @@ private void InitializeHttpFunctions(IEnumerable<FunctionDescriptor> functions,
 
                     WebJobsRouteBuilder builder = function.Metadata.IsProxy ? proxiesRoutesBuilder : routesBuilder;
                     builder.MapFunctionRoute(function.Metadata.Name, route, constraints, function.Metadata.Name);
+
+                    LogRouteMap(routesLogBuilder, function.Metadata.Name, route, httpTrigger.Methods, function.Metadata.IsProxy, _httpOptions.Value.RoutePrefix);
```

Comment:
nice that we're logging these now :)

Reply:
Yeah, thought I'd have that bonus here as it would have helped quite a bit.